src/Veldrid/Vk/StackList.cs

```diff
@@ -13,7 +13,6 @@ internal unsafe struct StackList<T> where T : struct
         public const int CapacityInBytes = 256;
         private static readonly int s_sizeofT = Unsafe.SizeOf<T>();
 
-        private fixed byte _storage[CapacityInBytes];
```

Comment:
Revert this: it completely breaks this type. It looks unused, but it's not.

Reply:
Fixed